src/client/transport/socketio.js

```diff
@@ -7,9 +7,10 @@
  */
 
 import * as ActionCreators from '../../core/action-creators';
-import io from 'socket.io-client';
 import { Transport } from './transport';
 
+const io = require('socket.io-client');
```

Comment:
What's the purpose of this change?

Reply:
The problem was that the socket.io-client library is not having a proper default export which  might fail in some nested bundling contexts. Didn't find a better way for that for now.

Reply:
I think this is also having the unintentional effect that `socket.io-client` no longer gets bundled into the minified bundle.  Try running `npm run build` with and without this PR you will notice a significant change in the bundle size.

Reply:
@ksjogo I think this is still not resolved.

Reply:
Sorry, missed the comment, will have a look.

Reply:
Changed it to keep the import syntax, builds fine with pnpm and npm now. And diff size seems roughly the same before/after.